bootstrap

```diff
@@ -4,32 +4,20 @@ set -e
 virtualenv="virtualenv"
 declare -a packages
 if [ -f /etc/debian_version ]; then
-    packages=(debianutils python-pip python-virtualenv python-dev libevent-dev libffi-dev libxml2-dev libxslt-dev zlib1g-dev)
+    packages=(debianutils python3-pip python3-virtualenv python3-dev libevent-dev libffi-dev libxml2-dev libxslt-dev zlib1g-dev)
     for package in ${packages[@]}; do
         if [ "$(dpkg --status -- $package 2>/dev/null|sed -n 's/^Status: //p')" != "install ok installed" ]; then
             # add a space after old values
             missing="${missing:+$missing }$package"
         fi
     done
+
     if [ -n "$missing" ]; then
         echo "$0: missing required DEB packages. Installing via sudo." 1>&2
         sudo apt-get -y install $missing
     fi
-else 
-    packages=(which libevent-devel libffi-devel libxml2-devel libxslt-devel zlib-devel)
-    if [ -f /etc/fedora-release ]; then
-        packages+=(python2-pip python2-virtualenv python2-devel)
-    elif [ -f /etc/redhat-release ]; then
-        unset ${GREP_OPTIONS}
-        eval $(cat /etc/os-release | grep VERSION_ID)
-        if [ ${VERSION_ID:0:1} -lt 8 ]; then
-            packages+=(python-virtualenv python-devel)
-        else
-            packages+=(python2-virtualenv python2-devel)
-            virtualenv="virtualenv-2"
-        fi
-    fi
-
+elif [ -f /etc/redhat-release ]; then
+    packages+=(which python3-virtualenv python3-devel libevent-devel libffi-devel libxml2-devel libxslt-devel zlib-devel)
     for package in ${packages[@]}; do
```

Comment:
The package python3-devel RPM name is python36-devel so it will fail the check in the next line

We can add the lines below to make an exception (it was tested only in centos 8:
```
        #When the package is python3-devel we change it to python36-devel
        if [ ${package} == "python3-devel" ]
        then
                package=python36-devel
        fi
```